test/integration/smoke/test_vm_snapshots.py

```diff
@@ -16,14 +16,16 @@
 # under the License.
```

Comment:
This file seems to be included by mistake. It already consists in this repo.
Please remove the relevant commit from PR and include only desired commit.